### PR TITLE
Clickable urls in list

### PR DESCRIPTION
**Pull request by yougotdirked**
- added simple <a> tag on icon that jumps to that topic.
- added hover styling on said icons.

### CODE REVIEW

File: build/style.css [hunk 1]
```diff
@@ -142,6 +143,11 @@ footer {
     align-items: center;
 }
 
+.topic-icon:hover {
+
```
Comment (drikusroor) [inline build/style.css:147]:
```suggestion
```

File: template.html [hunk 2]
```diff
@@ -71,7 +71,7 @@ <h2><a href="#infi-test">
             {{#each test.checks}}
             <li class="topic-{{topic}}">
               <div class="topic-icon {{topic}}">
-                <img src="/resources/{{topic}}.svg" alt="icon of {{topic}}" type="image/svg+xml">
+                <a href=#{{topic}}><img src="/resources/{{topic}}.svg" alt="icon of {{topic}}" type="image/svg+xml"></a>
               </div>
               <span>{{question}}</span>
```
Comment (drikusroor) [inline template.html:76]: I was wondering... Would it be nice if we could just click the whole line (icon + text) to jump towards that section, or why did you choose for the icon as a link specifically?

**Review by drikusroor — APPROVED** [review]
LGTM, I had a comment about whether or not it would be nice to also include the text into the anchor link, but I'll leave that up for you to decide.

**Comment by yougotdirked** [comment]
@drikusroor done and done

**Comment by drikusroor** [comment]
Something seems to be wrong with the [deploy preview](https://deploy-preview-82--the-infi-way.netlify.app/) or is that just on my machine? 🤔

**Pull request merged**